content/tutorial/03-sveltekit/01-concepts/01-introducing-sveltekit/README.md

```diff
@@ -1,37 +1,37 @@
 ---
-title: What is SvelteKit?
+title: SvelteKit là gì?
 ---
 
-Whereas Svelte is a _component framework_, SvelteKit is an _app framework_ (or 'metaframework', depending on who you ask) that solves the tricky problems of building something production-ready:
+Trong khi Svelte là một _framework component_ (bộ khung thành phần), SvelteKit là một _app framework_ (bộ khung ứng dụng, chứa các bộ khung thành phần) (hoặc 'metaframework', tùy thuộc vào người bạn hỏi) giải quyết những vấn đề khó khăn khi xây dựng một ứng dụng sẵn sàng để chạy thực tế (ứng dụng sẳn sàng tung ra thị trường cho người dùng sử dụng - production-ready):
```

Comment:
- Mình nghĩ không nên dịch các từ khóa này, nghe khá lạ và không truyền tải được rõ nghĩa lắm.
- mình đề xuất đổi cách hành văn một tí để tự nhiên và dễ hiểu hơn.

```suggestion
Nếu Svelte là một _framework component_ thì SvelteKit là một _app framework_ (hoặc 'metaframework' tùy vào cách định nghĩa). SvelteKit giúp giải quyết nhiều vấn đề phức tạp khi xây dựng một ứng dụng thực tế (production-ready):
```

Reply:
Mình có thể dịch app framework thành framework ứng dụng.

Reply:
Này mình search google và lấy định nghĩa google gợi ý từ các website khác. Ban đầu mình tính dẫn link search từ google cho người đọc click vào luôn cho tiện, nhưng nghĩ lại lại đưa vào luôn cho họ đọc luôn, vì xem bản dịch phần lớn là lười, k giỏi tiếng anh, tính tự tìm hiểu chưa cao hoặc k biết tìm sao. Suy nghĩ của mình là dịch cho người mới hoàn toàn hoặc chưa biết nhiều, chưa quen thuật ngữ công nghệ nhiều đọc, nên dịch và luôn để song ngữ phần từ khóa và giải thích định nghĩa từ khóa hoặc dẫn link nếu cần. Mọi người thấy sao.

Reply:
cái explanation này mình đọc cũng khá khó hiểu nên chắc để nguyên từ framework cho ăn chắc